tools/createRdFiles.R

```diff
@@ -191,16 +199,45 @@ pyVerbiageToLatex<-function(raw) {
 				substring(result, lcChar+1))
 	}
 
-	result<-gsub(dictDocString, "\nConstructor accepts arbitrary named arguments.\n", result, fixed=TRUE)
+	result<-gsub(dictDocString, "\nConstructor accepts named arguments.\n", result, fixed=TRUE)
 
-	convertSphinxToLatex(result)	
+	result<-convertSphinxToLatex(result)
+}
+
+getDescription<-function(raw) {
+	if (missing(raw) || is.null(raw) || length(raw)==0 || nchar(raw)==0) return("")
+	preprocessed<-gsub("\r\n", "\n", raw, fixed=TRUE)
+	# find everything up to the first double-newline
+	doubleNewLineIndex<-regexpr("\n\n", preprocessed)[1]
+	if (doubleNewLineIndex<=1) return("")
+	substr(preprocessed, 1, doubleNewLineIndex-1)
 }
 
 getReturned<-function(raw) {
-	NULL # TODO
+	if (missing(raw) || is.null(raw) || length(raw)==0 || nchar(raw)==0) return("")
```

Comment:
write a common function for getReturned and getExample, parameterized by regular expression.